.github/workflows/analysis.yml

```diff
@@ -12,6 +12,10 @@ on:
     paths-ignore:
       - "docs/**"
 
+concurrency: 
+  group: ${{ github.ref }}
```

Comment:
only scoped for the ref. should be enough right @paulgessinger ?

Reply:
This kills all workflows for the ref, right?

Reply:
yeah this is how I understand it